cmd/admin/main.go

```diff
@@ -70,9 +70,11 @@ func main() {
 
 	// This is the registry of all subcommands that the admin tool can run.
 	subcommands := map[string]subcommand{
-		"revoke-cert":  &subcommandRevokeCert{},
-		"block-key":    &subcommandBlockKey{},
-		"update-email": &subcommandUpdateEmail{},
+		"revoke-cert":   &subcommandRevokeCert{},
+		"block-key":     &subcommandBlockKey{},
+		"update-email":  &subcommandUpdateEmail{},
+		"pause-batch":   &subcommandPauseBatch{},
+		"unpause-batch": &subcommandUnpauseBatch{},
```

Comment:
nit: the other subcommands name the thing (cert, key, email) that they're affecting, not the quantity of things they're affecting. I would name these "pause" and "unpause", or "pause-ident" and "unpause-acct", or something like that.